src/shared/pages/team/teamView/teamView.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { Reducer, useEffect } from 'react';
+import { RouteComponentProps } from 'react-router';
+import { History } from 'history';
+import { deleteUserFromTeam, getTeamMembers } from '../../../services/usersService';
+import { getTeam } from '../../../services/teamsService';
+import { State } from './state';
+import { Action } from './actions';
+import { reducer } from './reducer';
+import { initialState } from './initialState';
+import { User } from '../../../models/user';
+import { GENERAL_ERROR_TITLE, LOAD_TEAM_ERROR_DESCRIPTION, LOAD_TEAM_MEMBERS_ERROR_DESCRIPTION } from '../../../models/constants';
+import ErrorSummary from '../../../common/components/errorSummary';
+
+interface MatchParams {
+    teamId: string;
+}
+
+interface TeamMembersProps extends RouteComponentProps<MatchParams> {
+    history: History;
```

Comment:
I think the RouteComponentProps interface already includes 'history'

Reply:
Yes it does. removed